tests/css_combinator/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -13,7 +13,7 @@ exports[`combinator.css 1`] = `
 .x
     .y {}
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
--Option/root .public/section ~ .public/section:before {
```

Comment:
@vjeux Do you happen to know where you copied this line from? Will stuff break for you if this is lowercased?

Reply:
I looked at both Facebook codebases that have CSS and this doesn't come up. Looking at the leading `-`, it's likely coming from some github diff that I copy and pasted and I forgot to remove the leading `-` for the first line. I don't remember where I found it, but I've also never seen leading `-` in CSS, so likely not an issue in practice.

Reply:
Found it, this was

```css
.AdsCMCampaignSummaryOption/root .public/AdsCMCampaignSummary/section ~ .public/AdsCMCampaignSummary/section:before {
```

that I tried to mangle. Definitely not going to break anything.